website/notifications/tasks.py

```diff
@@ -74,6 +74,7 @@ def _send_reviews_moderator_emails(send_type):
 
             mails.send_mail(
                 to_addr=user.username,
+                user=user,
```

Comment:
Drive-by: Was it just the user's `fullname` that was needed here? Because if so, that's available under `name` on L80.